scss/navCard.module.scss

```diff
@@ -55,12 +55,8 @@ $light_primary: color.adjust(variables.$primary, $lightness: 28%);
   font-size: 14px;
 }
 
-.lessons_tabsNav__nav__item--inactive:hover {
+.navCard__tabsNav__nav__item--inactive:hover {
   background-color: color.change(variables.$primary, $lightness: 95%);
   color: $dark_primary;
   transition: background-color 0.3s ease-out;
 }
-
-.lessons_tabs {
```

Comment:
This .lessons_tabs CSS class wasn't being used (https://github.com/garageScript/c0d3-app/search?q=lessons_tabs) so I deleted it.